Dockerfile

```diff
@@ -24,6 +24,16 @@ RUN if [[ $(uname -m) =~ "aarch64" ]]; then \
       pip install numcodecs; \
     fi
 
+ARG TRT_URL=http://cuda-repo.nvidia.com/release-candidates/Libraries/TensorRT/v10.3/10.3.0.25-4abf3f29/12.5-r555/Ubuntu22_04-x64-manylinux_2_17/deb/
```

Comment:
Hi @borisfom, I noticed that you've added this RC version of TensorRT. We're planning to update our base image to 24.08 in MONAI 1.4. I checked the TensorRT version in 24.08, and it seems to be newer than the one you installed. Do you think it will meet your requirements?
```
root@yunliu-ms-7d31:/workspace# dpkg -l | grep tensorrt
ii  tensorrt-dev                    10.3.0.26-1+cuda12.5                        amd64        Meta package for TensorRT development libraries
```

Reply:
Sure, is 24.08 out yet? Yes then we don't need to reinstall TRT.

Reply:
No, will be out by the end of Aug, I was using the staging one.

Reply:
@KumoLiu : Updated Dockerfile to use 24.08 base.